### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gndelia**
This PR revamps all the dependencies, with minimum support of node 12.  
At first, I only needed to update `web3-utils`, but given the other dependencies were just a few, and that the package is small, I updated all of them.

This is required to update `metronome-wallet-core`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,34 +19,34 @@
...
+    "eslint-config-bloq": "^2.4.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsdoc": "^35.4.1",
+    "eslint-plugin-mocha": "^9.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-require-path-exists": "^1.1.9",
-    "eslint-plugin-standard": "^4.0.0",
```
Comment (gndelia) [inline package.json:46]: this package was removed https://github.com/standard/standard/issues/1316

File: package.json [hunk 2]
```diff
@@ -19,34 +19,34 @@
   "main": "src/index.js",
   "repository": "autonomoussoftware/metronome-sdk-status",
   "scripts": {
-    "check": "dependency-check . && dependency-check . --unused --no-dev",
-    "check:outdated": "npm-check -i husky",
```
Comment (gndelia) [inline package.json:23]: `husky` is now in the latest version so no need to ignore it anymore (it's the one MIT licensed)

File: package.json [hunk 3]
```diff
@@ -19,34 +19,34 @@
...
+    "check": "dependency-check ./package.json --unused --no-dev",
```
Comment (gndelia) [inline package.json:22]: defaults have changed, so in order to check those prod deps not used, this is the proper script

File: tsconfig.json [hunk 4]
```diff
@@ -1,6 +0,0 @@
-{
-  "compilerOptions": {
-    "allowJs": true,
-    "noEmit": true
-  }
-}
```
Comment (gndelia) [inline tsconfig.json:6]: this file was not used (there's no TS!) so I removed it

**Review by gabmontes — DISMISSED** [review]

**Review by gabmontes — APPROVED** [review]

**Comment by gabmontes** [comment]
@gndelia package published manually as Travis disabled.

**Pull request merged**